### PR TITLE
Updated policy command

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (roneli) [inline .idea/modules.xml:1]: delete this file?

**Review by roneli — APPROVED** [review]
LGTM, had 1 comment, would probably add more docs/examples in the future but this is a good start

File: docs/cli/policy/overview.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+⚠️ Policy finished with warnings
+
+	✓   1.1  AWS CIS 1.1 Avoid the use of 'root' account. Show used in last 30 days (Scored)                                               passed
+
+	✓   1.2  AWS CIS 1.2 Ensure MFA is enabled for all IAM users that have a console password (Scored)                                     passed
+
+	❌  1.3  AWS CIS 1.3 Ensure credentials unused for 90 days or greater are disabled (Scored)                                            failed
+		❌  arn:aws:iam::XXXXXXXXXXXXXx:user/XXXXXXXXXXXXXx
+
+		❌  arn:aws:iam::XXXXXXXXXXXXXx:user/XXXXXXXXXXXXXx
```
Comment (bbernays) [inline docs/cli/policy/overview.md:65]: Can we document which fields get outputted?

File: docs/cli/policy/overview.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+
+	❌  1.4  AWS CIS 1.4 Ensure access keys are rotated every 90 days or less                                                              failed
+		❌  arn:aws:iam::XXXXXXXXXXXXXx:user/XXXXXXXXXXXXXx
+
+		❌  arn:aws:iam::XXXXXXXXXXXXXx:user/XXXXXXXXXXXXXXx
```
Comment (bbernays) [inline docs/cli/policy/overview.md:70]: What about output for manual queries?

Comment (yevgenypats) [reply docs/cli/policy/overview.md:70]: I tried to document it here - https://github.com/cloudquery/docs/pull/44/files#diff-737af9deaf1ee53eeae5d6bdb9c68f18ab2a6997c58e1a888be5ba7e3e159d17R40

Comment (bbernays) [reply docs/cli/policy/overview.md:70]:
```
 ⚠️  ELB-V2                                                                                                         manual
                  +----------------------------+-------------------------------------------------------------------------+--------------+-----------+ 
                  |            name            |                                dns_name                                 |  account_id  |  region   | 
                  +----------------------------+-------------------------------------------------------------------------+--------------+-----------+ 
                  | apigateway-nlb-integration | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.elb.us-east-1.amazonaws.com | xxxxxxxxxxxx | us-east-1 | 
                  +----------------------------+-------------------------------------------------------------------------+--------------+-----------+ 
                  | awseb-AWSEB-1Y07H683587FY  | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.us-east-1.elb.amazonaws.com         | xxxxxxxxxxxx | us-east-1 | 
                  +----------------------------+-------------------------------------------------------------------------+--------------+-----------+ 
                  | elbv2-integration          | xxxxxxxxxxxxxxxxxxxxxxxxxxx.us-east-1.elb.amazonaws.com                 | xxxxxxxxxxxx | us-east-1 | 
                  +----------------------------+-------------------------------------------------------------------------+--------------+-----------+ 
                             
```

File: docs/cli/policy/overview.md (hunk 2 above)
Comment (yevgenypats) [reply docs/cli/policy/overview.md:65]: Good note. cc @roneli @zagronitay , when something fails what is printed ?

Comment (roneli) [reply docs/cli/policy/overview.md:66]: When the query fails? I think ben means to document that we output identifier fields?

Comment (yevgenypats) [reply docs/cli/policy/overview.md:66]: got it. added.

**Pull request merged**